vision/common/box_plotter.py

```diff
@@ -41,10 +53,15 @@ def plot_box(boxes, image, waittime=0):
             X.append(x)
             Y.append(y)
 
-        cv2.rectangle(output, (min(X), min(Y)), (max(X), max(Y)), BBOX_COLOR, BBOX_THICKNESS)
-        #print('\tKeypoint:', bbox.pt[0], bbox.pt[1])
+        cv2.rectangle(
+            output, (min(X), min(Y)), (max(X), max(Y)), BBOX_COLOR, BBOX_THICKNESS
+        )
 
-        #output = cv2.drawKeypoints(image, keypoints, outImage=np.array([]), color=(255, 0, 255), flags=cv2.DRAW_MATCHES_FLAGS_DRAW_RICH_KEYPOINTS)
+        # print('\tKeypoint:', bbox.pt[0], bbox.pt[1])
+        # output = cv2.drawKeypoints(image, keypoints, outImage=np.array([]), color=(255, 0, 255), flags=cv2.DRAW_MATCHES_FLAGS_DRAW_RICH_KEYPOINTS)
```

Comment:
At a minimum we should add a `TODO` statement here along the lines of "Add boolean parameter to specify quiet output or not", similar to how it's done for benchmarking

Reply:
Alternatively, if you want to add a `quiet` parameter that defaults to `False`, it should be a pretty easy change